Makefile

```
@@ -126,27 +127,37 @@
 	go build -o bin/forklift-controller github.com/kubev2v/forklift/cmd/forklift-controller
 
+# Ensure temporary directories for forklift services
+.PHONY: ensure-temp-dirs
+ensure-temp-dirs:
+	install -d -m 700 /tmp/forklift-controller
+	install -d -m 700 /tmp/forklift-inventory
+
 # Run against the configured Kubernetes cluster in ~/.kube/config
 .PHONY: run
-run: generate fmt vet
+run: generate fmt vet ensure-temp-dirs
```

checkmake (0.2.2)

[warning] Target body for "run" exceeds allowed length of 5 (9).

(maxbodylength)